packages/react-app/src/App.jsx

```diff
@@ -113,40 +215,62 @@ function App({ mainnet, localProvider, appChainId }) {
       logoutOfWeb3Modal()
     })
 
-    console.log({ injectedProvider })
-    setTabValue(1)
+    // console.log({ injectedProvider })
+    setTabValue(prev => prev)
     // eslint-disable-next-line
-  }, [setInjectedProvider])
+  }, [appDispatch])
 
-  /* END - SETUP METAMASK */
+  // const loadWeb3ModalGoerli = useCallback(async () => {
```

Comment:
I think you can just remove the old code

Reply:
will do